test/parallel/test-windows-abort-exitcode.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+
+// This test makes sure that an aborted node process
+// exits with code 3 in windows.
+// Spawn a child, force an abort, and then check the
+// exit code in the parent. Assert on code and signal
+// to be of expected values.
+
+const spawn = require('child_process').spawn;
+if (!common.isWindows) {
+  common.skip('test is windows specific');
+  return;
+}
+if (process.argv[2] === 'child') {
+  process.abort();
+} else {
+  const child = spawn(process.argv[0], [process.argv[1], 'child']);
```

Comment:
Nit, but could you use `process.execPath` and `__filename` instead of the argv values? I think it's a little more readable.